src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
```

Comment:
Avoid using tag names for styling, except for `html` and `body`. Consider using a class or ID for styling the body element.